candle-pyo3/test.py

```diff
@@ -1,4 +1,5 @@
 import candle
+import torch
```

Comment:
Could we make a second test file specifically for the torch interaction? This would avoid requiring torch to be installed for running the main test.

Reply:
sure